api/src/shared/domain/errors.js

```diff
@@ -603,6 +603,12 @@ class AlreadyExistingInvitationError extends DomainError {
   }
 }
 
+class InvitationNotFoundError extends NotFoundError {
+  constructor(code = 'INVITATION_NOT_FOUND') {
+    super('Invitation not found', code);
+  }
+}
```

Comment:
Suggestion pour ne pas laisser penser qu’il est utile/légitime de spécifier un `code` lorsqu’on instancie une `InvitationNotFoundError` : 
```suggestion
class InvitationNotFoundError extends NotFoundError {
  constructor() {
    super('Invitation not found', 'INVITATION_NOT_FOUND');
  }
}
```